### PR TITLE
Add Command.cooldown

### PR DESCRIPTION
**Pull request by ImNimboss**
## Summary

Follow-up to #7721.

## Checklist

- [x] If code changes were made then they have been tested.
    - [x] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: discord/ext/commands/core.py [hunk 1]
```diff
@@ -664,6 +664,15 @@ def clean_params(self) -> Dict[str, inspect.Parameter]:
         """
         return self.params.copy()
 
+    @property
+    def cooldown(self) -> Optional[Cooldown]:
+        """Optional[:class:`Cooldown`]: The cooldown of a command when invoked or ``None`` if the command
+        doesn't have a registered cooldown.
```
Comment (Rapptz) [inline discord/ext/commands/core.py:670]:
```suggestion
        """Optional[:class:`~discord.app_commands.Cooldown`]: The cooldown of a command when invoked
        or ``None`` if the command doesn't have a registered cooldown.
```

**Review by Rapptz — COMMENTED** [review]
Good job, it's just failing the doc build due to a recent move of the `Cooldown` type.

**Comment by ImNimboss** [comment]
Thanks a lot for the help, fixed

**Review by Rapptz — APPROVED** [review]
:tada:

**Pull request merged**